google/resource_runtimeconfig_variable.go

```diff
@@ -0,0 +1,190 @@
+package google
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/runtimeconfig/v1beta1"
+	"regexp"
+)
+
+func resourceRuntimeconfigVariable() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceRuntimeconfigVariableCreate,
+		Read:   resourceRuntimeconfigVariableRead,
+		Update: resourceRuntimeconfigVariableUpdate,
+		Delete: resourceRuntimeconfigVariableDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"parent": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"project": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"value": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
add a ConflictsWith with the "text" field since only one or the other can be set.